src/mapping/grouping/attractor_mapping_featurizing.jl

```diff
@@ -95,18 +95,23 @@ ValidICS = Union{AbstractStateSpaceSet, Function}
 #####################################################################################
 # We only extend the general `basins_fractions`, because the clustering method
 # cannot map individual initial conditions to attractors
-function basins_fractions(mapper::AttractorsViaFeaturizing, ics::ValidICS;
-        show_progress = true, N = 1000
-    )
+function basins_fractions(mapper::AttractorsViaFeaturizing, og_ics::ValidICS;
+    show_progress = true, N = 1000, additional_ics::Union{ValidICS, Nothing} = nothing,
+)
+    if typeof(additional_ics) <: ValidICS
+        all_ics = deepcopy(og_ics)
+        append!(all_ics, additional_ics)
+        ics = all_ics
```

Comment:
How would this code work if `og_ics` is a function that spits out initial conditions...? `ValidICS` is either a function or a statespaceset.

Reply:
true, this is only useful for pre-generated `ics`. The line should be 
```julia
if typeof(additional_ics) <: StateSpaceSet
```